src/ComScoreStreamingAnalytics.ts

```diff
@@ -15,15 +13,23 @@ export class ComScoreStreamingAnalytics {
   private adBreakScheduleTime?: number;
   private logger: ComScoreLogger;
 
-  constructor(player: PlayerAPI, metadata: ComScoreMetadata = { mediaType: ComScoreMediaType.Other },
+  constructor(player: PlayerAPI, metadata: ComScoreMetadata = new ComScoreMetadata(),
               configuration: ComScoreConfiguration) {
-    this.logger = new ComScoreLogger(configuration);
```

Comment:
I would prefer the instance of the logger but no blocker.

Reply:
Just curious, why is this?